### DISCUSSION TITLE
Select from multiple objects

### DISCUSSION THREAD

**Discussion by CompuRoot** [TomWright/dasel#194] [Q&A] [answered]
Hi all !

I do probably something wrong, but I can't get  to work simplest things:
```
echo '{"name":"Bob"}{"name":"Tom"}' | dasel -m -p json '.name'
```
It returned:
```
Error: could not query multiple node: could not find multiple value: selector [type:PROPERTY selector:.name] does not support value: [kind:slice type:[]interface {}] [map[name:Bob] map[name:Tom]]
```

while the same things with `jq` works as expected:

```
echo '{"name":"Bob"}{"name":"Tom"}' | jq '.name'
```
which returns: 
```
"Bob"
"Tom"
```

I would be really appreciated if some one will find a time to explain what Im doing wrong

**Comment by TomWright** [comment, marked as answer]
You'll want to use `.[*].name` here.

`.name` is looking for a name property on the array, for which there isn't.
`[*]` performs the multi select for all items in that array, and then you can target the name of each.

https://dasel.tomwright.me/s/9be8b009-da16-4e97-97ee-b3be52547492

**Reply by CompuRoot** [reply]
Thank you ! I knew, it should be simple.
